docs/configuration.asciidoc

```diff
@@ -870,12 +870,13 @@ WARNING: We recommend always including the default set of validators if you cust
                                                                   "passwd",
                                                                   "pwd",
                                                                   "secret",
-                                                                  "*key",
+                                                                  "\*key",
                                                                   "*token*",
-                                                                  "*session*",
-                                                                  "*credit*",
-                                                                  "*card*",
-                                                                  "*auth*",
+                                                                  "\*session*",
+                                                                  "\*credit*",
+                                                                  "\*card*",
+                                                                  "\*auth*",
```

Comment:
Why this change? I assumed asciidoc must be consuming the `*` or something but it looks fine for me.

Reply:
yes, I realized in the preview that `*something*` is asciidoc for **bold**. Masking one of the `*` fixes that. For some reason, masking the `*` for `token` was not necessary...

Reply:
Ugh. Turns out the asciidoc preview in PyCharm renders things differently than our docs pipeline. I'll revert this change.